### ISSUE TITLE
Switch to using transient?

### ISSUE THREAD

**Issue by galeo** [joostkremers/pandoc-mode#97]
Thanks for this awesome package. 

Instead of using hydra, maybe it is more elegant to implement the UI with [transient](https://github.com/magit/transient).

**Comment by joostkremers** [comment]
Yes, it's something I've always wanted to look at. `transient` seems a much better fit because its use-case is exactly what `pandoc-mode` does: call a shell command with lots of different command line arguments.

The current hydra-based implementation has a big advantage (for me as the maintainer, I mean). If a new option or switch is added to `pandoc`, it usually takes only a single line of code to add support for it to `pandoc-mode`. It took a bit of work to set that up, but it works quite nicely, now. I will have to look at `transient` more closely to see if that can be made to work.

**Comment by galeo** [comment]
Hope to hear good news from you
soon. Thanks.
